requirements-vllm-cuda.txt

```diff
@@ -2,9 +2,4 @@
 # Dependencies for installing vLLM on CUDA
 
 # vLLM only supports Linux platform (including WSL)
-vllm>=0.5.0; sys_platform == 'linux' and platform_machine == 'x86_64' and python_version >= '3.9' and python_version <= '3.11'
-
-# Most the dependencies below were taken from https://github.com/vllm-project/vllm/blob/main/requirements-cuda.txt
```

Comment:
I believe these are still needed, why did they go away?

Reply:
vLLM's `setup.py` picks up the dependencies automatically: https://github.com/opendatahub-io/vllm/blob/main/setup.py#L367